extensions/ql-vscode/package.json

```diff
@@ -872,6 +877,13 @@
       }
     ],
     "menus": {
+      "editor/title": [
+        {
+          "command": "codeQL.runLocalQueryFromFileTab",
+          "group": "navigation",
+          "when": "config.codeQL.queriesPanel && resourceExtname == .ql && codeQL.currentDatabaseItem"
```

Comment:
Maybe more of a product question: Does this need to be feature-flagged behind `config.codeQL.queriesPanel`? 

It looks like this is a useful way to run queries, independent of the queries panel!

Reply:
I agree! Will discuss async